dbgmd-00001.sign

```diff
@@ -0,0 +1,6 @@
+tree 0003675ccdfb583bbe7bf1a3380aefaad9ce7c12
```

Comment:
Is this file something that should be ignored/removed?

Reply:
Sorry. That was a mistake.

I removed those files.